packages/@aws-cdk/cloudformation-include/README.md

```diff
@@ -274,7 +274,7 @@ and the nested stack in your CDK application as follows:
 ```typescript
 const parentTemplate = new inc.CfnInclude(this, 'ParentStack', {
   templateFile: 'path/to/my-parent-template.json',
-  nestedStacks: {
+  includeNestedStacks: {
```

Comment:
* overwrite
* replace
* hydrate
* reify
* load

Reply:
Does the same comment apply to the `includeNestedStack()` method? So, if I were to choose "overwrite", and would call this property `overwriteNestedStacks`, should I also change that method to be called `overwriteNestedStack()`?

Reply:
> * overwrite
> * replace
> * hydrate
> * reify
> * load

Also, out of curiosity. Why is `loadNestedStacks`, for example, better than `includeNestedStacks`, in your mind? The class and the module already use the word "include", I'm wondering what makes it not preferable in this one particular context.

Reply:
> Does the same comment apply to the includeNestedStack() method?

Yes they should be the same verb, otherwise it's extra-confusing.

> Why is "load" better than "include"? Why is "include" not preferable?

Hard to explain, and probably very subjective. To me, `include` means "add something in [implied: that wasn't there before]."

That's not what's happening here though. The nested stack was already there, but what we're doing is telling the construct a file on disk (`load`!) that it should grab for that nested stack. Furthermore, doing that is going to make the nested stack a different object (`replace`) than you would have gotten if you hadn't passed this nested stack. In some way, it's a more *full* (`hydrate`/`reify`) version of that nested stack.

All of that goes beyond "take a thing and add it in to what you already have". It's the connotation of *adding* that I have a problem with in this context.

`CfnInclude` is definitely an *include*. This... I don't necessarily agree.

Reply:
`loadNestedStack` feels right, it makes it clear that the action refers to to an existing nested stack and it also makes more sense that the user needs to supply a local path - "loading from disk". 
In  `CfnInclude` the "include" refers to "including a cloudFormation template in your CDK application", at least that how it sounds to me.

Reply:
Alright. "Load" it is!